docs/installation/installation.md

```diff
@@ -10,8 +10,9 @@
   - [Watching Namespaces](#watching-namespaces)
   - [Watching custom ingress class](#watching-custom-ingress-class)
   - [Installing cert-manager](#installing-cert-manager)
-- [Installing CRDs](#installing-crds)
-- [Installing Tyk Operator](#installing-tyk-operator)
+- [Installing Tyk Operator and CRDs](#installing-tyk-operator-and-crds)
+  - [Using Helm Chart](#helm-chart)
```

Comment:
It says `Using Helm Chart` here while heading of the section is `Helm Chart`. It should be the same.